teammapper-backend/src/map/services/maps.service.spec.ts

```diff
@@ -118,6 +118,15 @@ describe('MapsController', () => {
   })
 
   describe('deleteOutdatedMaps', () => {
+    it('deletes a map based off of lastAccessed', async () => {
```

Comment:
please also add test cases for these scenarios:

1) last access is set, but it should not be deleted due to more recent date
2) last access is not set at all, therefore old code should be used => lastModified
2a) lastModified too old => delete
2b) lastModified new => do not delete 
3) lastModified is too old but last access is rather new => should not be deleted
4) theoretical case, still might be sensible to test: last modified is new, lastAccessed too old => do not delete

Reply:
this is important because we have a running system with lots of data, would be rather unfortunate to loose all data ;)

Reply:
Good catch with point 3 and 4, both of these tests are failing right now :D Will fix tomorrow